packages/lit-html/src/lib/lit-html.ts

```diff
@@ -569,6 +582,28 @@ export class NodePart {
     public options: RenderOptions | undefined
   ) {}
 
+  detach(): NodePartState {
```

Comment:
Moving this out of the core would be ideal since it's only needed for some directives. 

Maybe the directive base class could have these methods and maybe that could be a separate import.

Reply:
Moving these to a separate module